download.py

```diff
@@ -5,33 +5,41 @@
 Originally created by kareemn on Jan. 1, 2019.
 https://github.com/kareemn/Tarteel-ML/blob/master/download.py
 
-Modified by Hamzah Khan (khanh111) and added to the tarteel.io/Tarteel-ML on Jan. 12.
+Modified by Hamzah Khan (khanh111) and added to the tarteel.io/Tarteel-ML on Jan. 12, 2019.
+
+Refactored completely by Hamzah Khan (khanh111) on Aug. 03, 2019.
 """
 
-import audio_preprocessing.recording_utils as recording_utils
+import utils.audio as audio_utils
+import utils.files as file_utils
+import utils.recording as recording_utils
 import csv
 import os
 import requests
 import wave
-import webrtcvad
 
 from urllib.parse import urlparse
 from argparse import ArgumentParser
 
 # Define argument constants.
 TARTEEL_V1_CSV_URL = "https://d2sf46268wowyo.cloudfront.net/datasets/tarteel_v1.0.csv"
 TARTEEL_LIVE_CSV_URL = "https://www.tarteel.io/download-full-dataset-csv"
+TARTEEL_V1_LABELED_CSV_URL = "https://tarteel-static.s3-us-west-2.amazonaws.com/labels/tarteel_v1.0_labeled.csv"
 
 
 parser = ArgumentParser(description='Tarteel Audio Downloader')
-parser.add_argument('--csv_url', type=str, default=TARTEEL_V1_CSV_URL)
-parser.add_argument('--local_csv_cache', type=str, default='.cache/local.csv')
-parser.add_argument('--local_download_dir', type=str, default='.audio')
-parser.add_argument('--vad_check', type=bool, default=True)
+parser.add_argument('--csv_url', type=str, default=TARTEEL_V1_LABELED_CSV_URL)
+parser.add_argument('--local_csv_filename', type=str, default='local.csv')
+parser.add_argument('--cache_dir', type=str, default='.cache')
+parser.add_argument('-u', '--use_cache', type=bool, default=False)
 parser.add_argument('-v', '--verbose', type=bool, default=False)
```

Comment:
We should use `store_true` here for --verbose so it doesn't require an extra argument from the command line

Reply:
Same for `--keep_downloaded_audio`

Reply:
Done. Should we keep downloaded audio by default? Can we listen to the raw files?

Reply:
If so, I'll change from `store_false` to `store_true` later.